config.py

```diff
@@ -14,10 +160,11 @@
 # DISEASE PARAMETERS
 AVG_INCUBATION_DAYS = 5 # &avg-incubation-days
 SCALE_INCUBATION_DAYS = 1
+INFECTIOUSNESS_ONSET_DAYS = 2.5 # relative to incubation days
```

Comment:
what's this? I don't see that it's used anywhere.

Reply:
It is used in `self.is_infectious`. This is based on the fact that infectiousness starts 2.5 days before the onset of the symptoms, and the time between exposure and symptoms onset is incubation days. so we have incubation days as a parameter and INFECTIOUSNESS_ONSET_DAYS is relative to it.  I wanted to go with self.viral_load > threshold, but it was not clear if it will fit with the time constraints of 2.5 days before symptom onset.